doc/website/blog/serverless-graphql-using-aws.md

```diff
@@ -0,0 +1,556 @@
+<div style={{textAlign: 'center'}}>
+  <img alt="Screenshot of AWS Lambda landing page listing functions" src="https://entgo.io/images/assets/appsync/function-overview.png" />
+  <p style={{fontSize: 12}}>AWS Lambda function overview of Ent function.</p>
+</div>
+Before reviewing the Go code and uploading the compiled Go code, we need to adjust some default settings of the function.
+First, we change the default handler name from `hello` to `main`, which equals the filename of the compiled Go binary:
+<div style={{textAlign: 'center'}}>
+  <img alt="Screenshot of AWS Lambda landing page listing functions" src="https://entgo.io/images/assets/appsync/runtime-settings.png" />
+  <p style={{fontSize: 12}}>AWS Lambda runtime settings of Ent function.</p>
+</div>
+Second, we add an environment the variable `DATABASE_URL` encoding the database network parameters and credentials:
+<div style={{textAlign: 'center'}}>
+  <img alt="Screenshot of AWS Lambda landing page listing functions" src="https://entgo.io/images/assets/appsync/envars.png" />
+  <p style={{fontSize: 12}}>AWS Lambda environemnt variables settings of Ent function.</p>
+</div>
+The database URL follows the schema `postgres://username:password@hostname/dbname`.
+By default, AWS Lambda encrypts the environment variables, making them a fast and safe mechanism to supply database connection parameters.
+Alternatively, one can supply an ARN ID to an AWS Secretsmanager secret from which the function requests new credentials.
+Using AWS Secretmanager supports rotating credentials but requires an additional request at the start of each Lambda.
+A third option is to use AWS IAM to handle the database authorization.
+The former two options make only sense when using AWS RDS.
+
+If you created your Postgres database in AWS RDS, the default username and database name are `postgres`.
+The password can be reset by modifying the AWS RDS instance.
+Be aware that AWS RDS is a rather expensive service and that the network configuration is highly non-trivial!
+
+### Setting up Ent and deploying AWS Lambda
+
+We now review, compile and deploy the missing Go program, which you can also find at [bodokaiser/entgo-aws-appsync](https://github.com/bodokaiser/entgo-aws-appsync), for the "ent" function to run.
+
+First, we create an empty directory to which we change:
+```console
+mkdir entgo-aws-appsync && cd $_
+```
+Second, we enable go modules and install the Ent toolset:
+```console
+go mod init entgo-aws-appsync
+go mod tidy
+go get -d entgo.io/ent/cmd/ent
+```
+Third, we create the `Todo` schema
+```console
+go run entgo.io/ent/cmd/ent init Todo
+```
+and add the `title` field:
+```go title="ent/schema/todo.go"
+package schema
+
+import (
+	"entgo.io/ent"
+	"entgo.io/ent/schema/field"
+)
+
+// Todo holds the schema definition for the Todo entity.
+type Todo struct {
+	ent.Schema
+}
+
+// Fields of the Todo.
+func (Todo) Fields() []ent.Field {
+	return []ent.Field{
+		field.String("title"),
```

Comment:
Use the [Line Highlighting](https://docusaurus.io/docs/markdown-features/code-blocks#line-highlighting) feature of mardown to highlight the changes: